.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,138 @@
+---
+name: Bug Report
+description: Create a report to help us improve.
+labels: [bug]
+assignees: aio-libs/triagers
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Thanks for taking a minute to file a bug report!**
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+      _Please fill out the form below with as many precise
+      details as possible._
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >-
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    value: |
```

Comment:
Maybe change this to a placeholder?

Reply:
Maybe change it to something like this:
```
1. Implement the following server or a client '...'
2. Then run '...'
3. An error occurs.
```

Reply:
Yes, please.

Reply:
This is still a value. We agreed on placeholder earlier.

Reply:
Sorry my bad. I will make the changes.